grafana-altera/public/assets/fonts/FontAwesome/LICENSE.txt

```diff
@@ -0,0 +1,34 @@
+Font Awesome Free License
```

Comment:
FontAwesome có thể được install bằng yarn hoặc npm `npm install --save-dev @fortawesome/fontawesome-free` https://www.angularjswiki.com/angular/how-to-use-font-awesome-icons-in-angular-applications/